src/main/java/FigureSupplier.java

```diff
@@ -0,0 +1,23 @@
+import java.util.Random;
+
+public class FigureSupplier {
+    public Figure getRandomFigure() {
+        String[] nameFigure = new String[]{"circle", "square", "rectangle",
```

Comment:
let's make figureNames constant
```suggestion
        String[] figureNames = new String[]{"circle", "square", "rectangle",
```